### ISSUE TITLE
Add SEMoS 8.2.11 Categories to SE:Vocab

### ISSUE THREAD

**Issue by JenRodge** [standardebooks/web#141]
From the [Standard Ebooks Manual of Style, 8.2.11.2](https://standardebooks.org/manual/1.6.1/8-typography#8.2.11.2)
> The names of publications, music, and art that can stand alone are italicized; additionally, the names of transport vessels are italicized. These include, but are not limited to:⁠ ⁠…

It seems reasonable that all of the items listed to be italicized also appear in the [SE vocabulary](https://standardebooks.org/vocab/1.0) so that they can be semanticated (assuming they're not in EPUB® 3 or Z39.98), but there are a few missing: `se:name.music.album`, `se:broadcast.radio-show`, and `se:name.publication.ballad` (or possibly `se:name.music.ballad`).

Further, some clarification may be in order.  [8.2.11.3](https://standardebooks.org/manual/1.6.1/8-typography#8.2.11.3) lists names that should be enquoted, but there is no indication if they should be semanticated or what that would look like (`“<span epub:type="se:name.music.song">Happy Birthday</span>”`?).  Some of these listed enquotable names (song, short-story, novella) are in the vocabulary, but not all.

In short, three proposals:
1. Add all 8.2.11.2 names to the SE vocabulary
2. Specify in the SEMoS if enquoted names should be semanticated
3. If yes to proposal 2, add 8.2.11.3 items to the SE vocabulary

**Comment by acabal** [comment]
Most of the things listed, albums, radio shows, etc., don't occur often enough in our pre-1925 corpus to make it worth it. Usually if it's not in the vocabulary, you can simply omit the more specific part of the semantic, for example `<i epub:type="se:name.music">`. Ballads in particular are types of poetry so they get `se:name.publication.poem`.

Re. semantics on things in quotes, that is really the old problem of "how far do we go with semantics?" And the answer is, if you're going to put a tag there anyway, like for italics, then add semantics. Otherwise, adding further semantics with things like `<span>` is up to your discretion. One can go down a real rabbit hole with TEI-style semantics which not only is a huge time sink but also makes little sense for 99% of use cases.
